packages/react-strict-dom/src/native/index.js

```diff
@@ -56,4 +50,11 @@ const contexts = {
 // Export using StyleX types as the shim has divergent types internally.
 const css: TStyleX = _css as $FlowFixMe;
 
+export type { StaticStyles };
+export type { StyleXStyles as Styles };
+export type { StyleXStylesWithout as StylesWithout };
+export type { Theme as StyleTheme };
+export type { Types as StyleTypes };
+export type { VarGroup as StyleVars };
```

Comment:
1. The exports need to be updated in the DOM branch too.
2. Make it easy to drop the `Types` export during internal sync